packages/jaeger-ui/src/model/ddg/transformDdgData.tsx

```diff
@@ -18,36 +18,36 @@ import {
   TDdgPayload,
   TDdgPayloadEntry,
   TDdgPath,
-  TDdgPathElemsByDistance,
+  TDdgDistanceToPathElems,
   TDdgServiceMap,
   TDdgVisibilityIdxToPathElem,
 } from './types';
 
-const stringifyPayloadEntry = ({ service, operation }: TDdgPayloadEntry) => `${service}\v${operation}`;
+const stringifyEntry = ({ service, operation }: TDdgPayloadEntry) => `${service}\v${operation}`;
 
 export default function transformDdgData(
   payload: TDdgPayload,
   { service: focalService, operation: focalOperation }: { service: string; operation?: string }
 ): TDdgModel {
   const serviceMap: TDdgServiceMap = new Map();
-  const pathElemsByDistance: TDdgPathElemsByDistance = new Map();
-  const pathsComparisonMap: Map<TDdgPayloadEntry[], string> = new Map();
+  const distanceToPathElems: TDdgDistanceToPathElems = new Map();
+  const pathsComparators: Map<TDdgPayloadEntry[], string> = new Map();
 
   const paths = payload
     .slice()
     .sort((a, b) => {
-      let aCompareValue = pathsComparisonMap.get(a);
-      if (!aCompareValue) {
-        aCompareValue = a.map(stringifyPayloadEntry).join();
-        pathsComparisonMap.set(a, aCompareValue);
+      let aComparator = pathsComparators.get(a);
```

Comment:
What was wrong with `aCompareValue` (aside from length)?

IIRC, a comparator is a thing that compares, e.g. a function or something with a `comparator.compare(a, b)` method?

Reply:
reverted to `aCompareValue`
I thought comparator was akin to measure as both a verb and a noun and was looking to shorten...